strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/configuration/ConfigurationManagerTest.java

```diff
@@ -66,21 +67,40 @@
 
     private YAMLMapper yamlMapper;
 
-
     @BeforeEach
     public void setUp()
             throws IOException
     {
-        Path yamlPath = Paths.get(CONFIGURATION_BASEDIR);
-        if (Files.notExists(yamlPath))
+        if (Files.notExists(CONFIGURATION_BASEDIR))
         {
-            Files.createDirectories(yamlPath);
+            Files.createDirectories(CONFIGURATION_BASEDIR);
         }
 
         yamlMapper = yamlMapperFactory.create(
                 Sets.newHashSet(CustomRepositoryConfigurationDto.class, RemoteRepositoryConfigurationDto.class));
     }
 
+    @AfterAll
```

Comment:
will be better to use `@AfterEach` with `TestInfo` instead of static `@AfterAll`